api/src/apps/ovfiets/importer.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+from django.contrib.gis.geos import Point
+from django.db.models import FieldDoesNotExist
+from django.utils import timezone
+
+from apps.base.models import District, Neighbourhood
+from apps.ovfiets.models import OvFiets, OvFietsRaw
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+
+class OvFietsImporter:
+    '''
+    Class used to unpack OvFietsRaw records, clean them then
+    insert them into OvFiets model
+
+    altered_keys is a dictionary containing the keys in the raw instances
+    that need to be altered to the model field
+    '''
+    altered_keys = {
+        'openingHours': 'opening_hours',
+        'locationCode': 'location_code',
+        'stationCode': 'station_code',
+        'fetchTime': 'fetch_time',
+        'rentalBikes': 'rental_bikes'
+    }
+
+    def clean_key(self, key):
+        '''
+        Checks if key needs to be altered
```

Comment:
"Checks if the csv key needs to be mapped to a model attribute with a different name."